### ISSUE TITLE
FocusTrap not removing aria-hidden attribute

### ISSUE THREAD

**Issue by Keilo75** [mantinedev/mantine#2726]
### What package has an issue

@mantine/hooks

### Describe the bug

Just found a bug which appears to affect all components using FocusTrap - Modal, Drawer, etc..
If you turn the focus trap on, the root gets hidden using "aria-hidden"="true", which makes sense. But this attribute is never removed and even when the focus trap is removed, the content remains inaccessible.

### What version of @mantine/hooks page do you have in package.json?

^5.5.2

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/hopeful-black-jopb3u?file=/src/App.js

### Do you know how to fix the issue

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

I *believe* this is caused by https://github.com/mantinedev/mantine/blob/9128d03970df766429c926c000c29e8c225e6c33/src/mantine-hooks/src/use-focus-trap/create-aria-hider.ts#L32, as the aria hidden only gets removed if it is already null.

If that is the case, I'd be willing to fix it. I have never contributed to an open-source project of this scale before, so it might take some time to figure things out - hope that'd be okay. But given the fact that this file was last changed in april of 2021, I don't reckon it's a high priority fix

**Comment by rtivital** [comment]
Hi @Keilo75, feel free to send a PR
